src/main/java/uk/gov/companieshouse/web/accounts/transformer/smallfull/intangible/impl/IntangibleAssetsTransformerImpl.java

```diff
@@ -7,19 +7,20 @@
 import org.springframework.stereotype.Component;
 import uk.gov.companieshouse.api.model.accounts.smallfull.intangible.IntangibleApi;
 import uk.gov.companieshouse.web.accounts.enumeration.IntangibleAssetsResource;
+import uk.gov.companieshouse.web.accounts.enumeration.NoteType;
 import uk.gov.companieshouse.web.accounts.model.smallfull.notes.intangible.IntangibleAssets;
+import uk.gov.companieshouse.web.accounts.transformer.NoteTransformer;
 import uk.gov.companieshouse.web.accounts.transformer.smallfull.intangible.IntangibleAssetsResourceTransformer;
-import uk.gov.companieshouse.web.accounts.transformer.smallfull.intangible.IntangibleAssetsTransformer;
 import uk.gov.companieshouse.web.accounts.transformer.smallfull.intangible.IntangibleAssetsTransformerFactory;
 
 @Component
-public class IntangibleAssetsTransformerImpl implements IntangibleAssetsTransformer {
+public class IntangibleAssetsTransformerImpl implements NoteTransformer<IntangibleAssets, IntangibleApi> {
 
     @Autowired
     private IntangibleAssetsTransformerFactory factory;
 
     @Override
-    public IntangibleAssets getIntangibleAssets(IntangibleApi intangibleApi) {
+    public IntangibleAssets toWeb(IntangibleApi intangibleApi) {
```

Comment:
We need an additional check in this method for the case where `intangibleApi == null`

Reply:
> We need an additional check in this method for the case where `intangibleApi == null`

... which would then return null? Or just empty?

Reply:
If the api object is null, we need to return an 'empty' object to back the page - see other transformers for examples 👍

Reply:
Fixed.